avm/client/src/avmHelpers.ts

```diff
@@ -0,0 +1,129 @@
+import { CallResultsArray, InterpreterResult, CallRequest } from './types';
+
+const decoder = new TextDecoder();
+const encoder = new TextEncoder();
+
+export function prepareArgs(
+    air: string,
+    prevData: Uint8Array,
+    data: Uint8Array,
+    params: {
+        initPeerId: string;
+        currentPeerId: string;
+    },
+    callResults: CallResultsArray,
+): string {
+    const callResultsToPass: any = {};
+    for (let [k, v] of callResults) {
+        callResultsToPass[k] = {
+            ret_code: v.retCode,
+            result: v.result,
+        };
+    }
+
+    const paramsToPass = {
+        init_peer_id: params.initPeerId,
+        current_peer_id: params.currentPeerId,
+    };
+
+    const encoded = encoder.encode(JSON.stringify(callResultsToPass));
+
+    const avmArg = JSON.stringify([
+        // force new line
```

Comment:
what does it mean?

Reply:
It forces prettier (the de-facto standard JS formatting tool) to force print new line here :)

Reply:
Got it, thank you